### ISSUE TITLE
"aria-live" is not read if we use portable version

### ISSUE THREAD

**Issue by stanislavhacker** [nvaccess/nvda#10510]
### Steps to reproduce:
Start nvda as portable
Open chrome 78
Go to: http://accessibility.athena-ict.com/aria/examples/live3.shtml
Try "set alert" with 10s option

### Actual behavior:
After alert show, nvda nothing read

### Expected behavior:
After alert show, nvda read this text in alert

**Comment by Adriani90** [comment]
cc: @derekriemer

**Comment by Adriani90** [comment]
@stanislavhacker are you having this issue also with NVDA 2024.1 RC1?

**Comment by Adriani90** [comment]
In NVDA 2024.2 and Firefox 127 or Chromium browsers, NVDA does not read aria live allert on the "set rude allert button" with aria-live="assertive" at all. @jcsteh any reason why this does not work?
cc: @SaschaCowley. 

Note: this issue is not related to portable copy, it happens with installed copy as well.

**Comment by Adriani90** [comment]
Closing as duplicate of #6335.
